Resources/doc/events.md

```diff
@@ -8,14 +8,14 @@ FSiAdminSecurityBundle provides some useful events which cause some default acti
 user instance. Its build-in subscribers:
```

Comment:
"*is fired when new password is set on user instance. Its build-in subscribers: " => "is fired when a new password is set on user's instance. It has following subscribers built-in :"